<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuDialogUI</class>
 <widget class="QWidget" name="MuDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="MuBaseWidget" name="dlgCentralWidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="MuDialogTitleWidget" name="dlgTitleWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="dlgTitleLb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>MuDialog</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MuDialogCloseButton" name="dlgCloseBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="dlgMainWidget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="dlgBottomWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="dlgButton1">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Button1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dlgButton2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>Button2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MuBaseWidget</class>
   <extends>QWidget</extends>
   <header>MuBaseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MuDialogTitleWidget</class>
   <extends>QWidget</extends>
   <header>MuDialogUI.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MuDialogCloseButton</class>
   <extends>QPushButton</extends>
   <header>MuDialogUI.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
